fil_actor_interface/Cargo.toml

```diff
@@ -29,6 +29,7 @@ fvm_ipld_encoding.workspace = true
 fvm_shared = { workspace = true }
 fvm_shared3 = { workspace = true }
 lazy_static.workspace = true
+multihash = { workspace = true, features = ["identity"] }
```

Comment:
Let's change `known_cids` to import from `multihash` directly rather than through `Cid`. That way, it is easier to see why we have this dependency.